### ISSUE TITLE
Colour contrast of search field input on news / directories etc

### ISSUE THREAD

**Issue by msayoung** [localgovdrupal/localgov_base#642]
In localgov_base our search field input does not have enough contrast between itself and the background. 

- The  --input-bg-color is white
- The input border colour / --input-border-color is #b1b4b6
- The background / --sidebar-exposed-form-bg-color  #cecfd0

One of those needs to have 3:1 contrast with eachother. I suggest we change the --input-border-color to black. 

More info here: https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html
